core/src/main/java/org/pentaho/platform/scheduler2/ISchedulerOutputPathResolver.java

```diff
@@ -1,7 +1,7 @@
 /*!
  * HITACHI VANTARA PROPRIETARY AND CONFIDENTIAL
  *
- * Copyright 2023 Hitachi Vantara. All rights reserved.
+ * Copyright 2023-2024 Hitachi Vantara. All rights reserved.
```

Comment:
Apparently, no changes needed on this file.

Reply:
fixed